core/components/minishop2/handlers/msorderhandler.class.php

```diff
@@ -452,8 +452,7 @@ public function getCost($with_cart = true, $only_cost = false)
                 ['id' => $this->order['delivery']]
             )
         ) {
-            $cost = $delivery->getCost($this, $cost);
-            $delivery_cost = $cost - $cart['total_cost'];
+            $cost += $delivery_cost =  $delivery->getCost($this, $cost) - $cost;
```

Comment:
```suggestion
            $cost += $delivery_cost = $delivery->getCost($this, $cost) - $cost;
```

тут у тебя лишний пробел и разделяй PR, почему всё намешано. Тем более без какого-либо описания.